services/controllerhost/queueDepthCache.go

```diff
@@ -96,9 +95,6 @@ func (cache *storeExtentMetadataCache) fetchStoreExtentMetadata(extID string, st
 	}
 
 	rs := stats.GetReplicaStats()[0]
-	// store apparently reports illegal values
-	// fix them if needed
-	cache.fixReplicaStatsIfBroken(rs)
```

Comment:
still want to leave this around to be backward compatible.. We should be able to delete this code once all the data has been touched by your new reporting.